<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>置換</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/magnifier--pencil.png</normaloff>:/images/magnifier--pencil.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="6">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="findPrev">
       <property name="text">
        <string>上検索(&amp;U)</string>
       </property>
       <property name="shortcut">
        <string>Shift+Return</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findNext">
       <property name="text">
        <string>下検索(&amp;D)</string>
       </property>
       <property name="shortcut">
        <string>F3</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findMark">
       <property name="text">
        <string>該当マーク(&amp;B)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="replace">
       <property name="text">
        <string>置換(&amp;R)</string>
       </property>
       <property name="shortcut">
        <string>F3</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAll">
       <property name="text">
        <string>全置換(&amp;A)</string>
       </property>
       <property name="shortcut">
        <string>F3</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit">
       <property name="text">
        <string>キャンセル(&amp;X)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="4" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>検索オプション</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="caseSensitively">
        <property name="text">
         <string>大文字と小文字を区別する(&amp;C)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="wholeWords">
        <property name="text">
         <string>単語単位(&amp;W)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="regularExpression">
        <property name="text">
         <string>正規表現(&amp;R)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="findkeep">
        <property name="text">
         <string>検索後ダイアログを閉じる</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QCheckBox" name="warningNavi">
        <property name="text">
         <string>見つからないときに警告を表示する</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>マークする(&amp;M)</string>
          </property>
          <property name="buddy">
           <cstring>mark</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="mark">
          <property name="whatsThis">
           <string>検索結果の出力を「Grepウィンドウ」、「Grepエディタ」形式か選択できます</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="1">
      <widget class="QComboBox" name="before">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxCount">
        <number>20</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>置換前(&amp;N):</string>
       </property>
       <property name="buddy">
        <cstring>before</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="after">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxCount">
        <number>20</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>置換後(&amp;P):</string>
       </property>
       <property name="buddy">
        <cstring>after</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="nextSearch">
     <property name="text">
      <string>置換後、次の文字列を検索する</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
